### ISSUE TITLE
Improve UI

### ISSUE THREAD

**Issue by darthgera123** [darthgera123/QuizApp#2]
The current UI is very bland and some major color changes are required.

**Comment by NirmalKumarJoshi** [comment]
Hello @darthgera123  can I work on this under GSSOC'19?

**Comment by darthgera123** [comment]
yeah sure. However, keep in mind that contributions will be counted from March 1st i think. Meanwhile, play around with the project.

**Comment by 9267aakashsharma** [comment]
I also want to contribute to it under GSSOC'19. I had building UI and frontend for website and am looking forward to work with this one too.

**Comment by adhavalboy** [comment]
@9267aakashsharma Thanks for your interest go through issues or if you have new idea you can implement, I am one of mentor this project if you want any help you can ping on slack.

**Comment by Jayashree-panda** [comment]
@darthgera123 hello. I am interested for improving the UI of homepage under gssoc 2019. Can I?

**Comment by Rohithgilla12** [comment]
Yes any of you please go ahead

**Comment by 9267aakashsharma** [comment]
I have my internal exams, will start the work right after holi.
